### ISSUE TITLE
fetchClosedOrder - bug

### ISSUE THREAD

**Issue by BHruska** [ccxt/ccxt#7887]
Trying to fetch closed orders on Coinbase Pro.  Worked fine forever, but now I'm getting this error.  I've updated to the latest and greatest, but still crashes here.  It does work fine on Kraken and other exchanges.

Ideas?

Thanks,
Brian

- OS: WINDOWS 10
- Programming Language version: PYTHON 3.6
- CCXT version:  CCXT Version: 1.37.3

```
REPLACE_THIS_WITH_YOUR_CODE_TO_REPRODUCE_THE_ISSUE_WITHOUT_YOUR_KEYS
```
    def getOrdersClosed(self, symbol, since=0):
        if since == 0:
            since = self.ordersFilterDays
        if not self.validatePair(symbol):
            print('getOrdersClosed: Invalid Pair')
            return []
        if not self.exchange.has['fetchClosedOrders']:
            print('getOrdersClosed: No fetchClosedOrders')
            return []         
        return self.exchange.fetchClosedOrders(symbol, since)

```
C:\PythonProjects\ccxt>C:/Python36/python.exe c:/PythonProjects/ccxt/getExchangeOrdersClosed.py
Traceback (most recent call last):
  File "c:/PythonProjects/ccxt/getExchangeOrdersClosed.py", line 29, in <module>
    main()
  File "c:/PythonProjects/ccxt/getExchangeOrdersClosed.py", line 26, in main
    getExchangeOrdersOpen()
  File "c:/PythonProjects/ccxt/getExchangeOrdersClosed.py", line 12, in getExchangeOrdersOpen
    orders = exchange.getOrdersClosed(pair)
  File "c:\PythonProjects\ccxt\_EXCHANGE.py", line 499, in getOrdersClosed
    return self.exchange.fetchClosedOrders(symbol, since)
  File "C:\Python36\lib\site-packages\ccxt\coinbasepro.py", line 771, in fetch_closed_orders
    return self.parse_orders(response, market, since, limit)
  File "C:\Python36\lib\site-packages\ccxt\base\exchange.py", line 1714, in parse_orders
    return self.filter_by_symbol_since_limit(array, symbol, since, limit)
  File "C:\Python36\lib\site-packages\ccxt\base\exchange.py", line 1771, in filter_by_symbol_since_limit
    return self.filter_by_value_since_limit(array, 'symbol', symbol, since, limit)
  File "C:\Python36\lib\site-packages\ccxt\base\exchange.py", line 1765, in filter_by_value_since_limit
    array = [entry for entry in array if entry[key] >= since]
  File "C:\Python36\lib\site-packages\ccxt\base\exchange.py", line 1765, in <listcomp>
    array = [entry for entry in array if entry[key] >= since]
TypeError: '>=' not supported between instances of 'NoneType' and 'int'
```

**Comment by kroitor** [comment]
@BHruska 

- https://github.com/ccxt/ccxt/wiki/FAQ#what-is-required-to-get-help

Can you please post your verbose output? What's your value of `since` ?

```
self.exchange.load_markets()
self.exchange.verbose = True
return self.exchange.fetch_closed_orders(symbol, since)
```

**Comment by FishLiuabc** [comment]
Same issues happened to me, any fixes?

**Comment by kroitor** [comment]
@FishLiuabc please, post the following info (all of it) here: https://github.com/ccxt/ccxt/wiki/FAQ#what-is-required-to-get-help

**Comment by jamie0xgitc0decat** [comment]
I got the same issue. It looks like ccxt didn't get timestamp value from the order information.

Exchange: Bybit
CCXT version: 1.73.12

```python
if (bybit_client.has['fetchClosedOrders']):
    start_date_ts = datetime.datetime(2020,5,1).replace(tzinfo=timezone.utc).timestamp()
    print(start_date_ts)
    res = bybit_client.fetchClosedOrders (symbol = "BTC/USDT", since = start_date_ts, limit = 50, params = {""})
    print(res[0])
    print([record["timestamp"] for record in res])
```

```
{'info': {'order_id': 'xxxxxx', 'user_id': 'xxxxxx', 'symbol': 'BTCUSDT', 'side': 'Sell', 'order_type': 'Market', 'price': '41833.5', 'qty': '0.1', 'time_in_force': 'ImmediateOrCancel', 'order_status': 'Filled', 'last_exec_price': '44035', 'cum_exec_qty': '0.1', 'cum_exec_value': '4403.5', 'cum_exec_fee': '3.302625', 'reduce_only': True, 'close_on_trigger': True, 'order_link_id': 'xxxxxxxxx', 'created_time': '2022-02-14T07:28:20Z', 'updated_time': '2022-02-14T07:28:20Z', 'take_profit': '0', 'stop_loss': '0', 'tp_trigger_by': 'UNKNOWN', 'sl_trigger_by': 'UNKNOWN'}, 'id': 'xxxxxxx', 'clientOrderId': 'xxxxxxxxxx', 'timestamp': None, 'datetime': None, 'lastTradeTimestamp': None, 'symbol': 'BTC/USDT', 'type': 'market', 'timeInForce': 'IOC', 'postOnly': False, 'side': 'sell', 'price': 44035.0, 'stopPrice': None, 'amount': 0.1, 'cost': 4403.5, 'average': 44035.0, 'filled': 0.1, 'remaining': 0.0, 'status': 'closed', 'fee': {'cost': 3.302625, 'currency': 'USDT'}, 'trades': [], 'fees': [{'cost': 3.302625, 'currency': 'USDT'}]}


[None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None]
```

```
https://github.com/ccxt/ccxt/blob/master/python/ccxt/base/exchange.py

![image](https://user-images.githubusercontent.com/32829756/154194757-bef31d70-c261-4baf-aa95-851b4172162a.png)

```
I changed the line 2072 to the following to resolve this issue.
array = [entry for entry in array if datetime.datetime.strptime(entry["info"]["created_time"],"%Y-%m-%dT%H:%M:%SZ").timestamp() >= since]
```

**Comment by FishLiuabc** [comment]
> @FishLiuabc please, post the following info (all of it) here: https://github.com/ccxt/ccxt/wiki/FAQ#what-is-required-to-get-help

Thanks for the suggestions and sorry for the late reply, I've submitted an issue. here: https://github.com/ccxt/ccxt/issues/12032
